dist/common/scripts/node_exporter_install

```diff
@@ -20,15 +20,13 @@
 # You should have received a copy of the GNU General Public License
 # along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
 
-import os
-import sys
-import tempfile
 import tarfile
-from scylla_util import *
 import argparse
 
-VERSION='0.17.0'
-INSTALL_DIR=scylladir()+'/Prometheus/node_exporter'
+from scylla_util import *
+
+VERSION = '0.17.0'
+INSTALL_DIR = scylladir()+'/Prometheus/node_exporter'
 
```

Comment:
Is this related?

Reply:
It's adding spaces to conform to PEP8
And remove already imported packages, since the star import from scylla_utils already import those.

Reply:
The subject is "Adding pylint execution"

This isn't adding pylint execution.